### ISSUE TITLE
Accessibility: Books list is extremely hard to navigate

### ISSUE THREAD

**Issue by Menelion** [audiamus/BookLibConnect#170]
Your AaxAudioconverter app is extremely accessible for screen reader users due to windowsForms native accessibility. Unfortunately, this is not the case with BookLibConnect.  
For some reason, the book list in the library is almost impossible to navigate, instead of every list item (which should be a book title) "system.Drawing.Bitmap" is reported to screen readers.  
I'm now myself into Windowsforms development, but I had only like half an hour to investigate so far. I'd be glad to submit a pull request but I need to familiarize myself with the code. If you guide me where to dig, it would be super kind of you.

**Comment by audiamus** [comment]
Can you provide a screenshot?

**Comment by Menelion** [comment]
I can but how would it help? I'm talking about [screen readers](https://en.wikipedia.org/wiki/Screen_reader) for blind users.

**Comment by audiamus** [comment]
I think I understand now. It's not the actually rendering of the form (although I have seen weird things there before), but the way an external application may analyse it. The first column in both table windows contains an icon, which is presumably rendered as Bitmap. You would want a version with no icons? A very special case, I'm afraid, so I suggest to modify the code for yourself.

**Comment by Menelion** [comment]
Aha, understood. No, I don't want a special version of course. But can we mark those icons up with, say, `AccessibleDescription` or something like that? It would be ideal to have this column report book title instead. Anyway, could you please point me where it is in the code? Thanks!

**Comment by audiamus** [comment]
The DataGridViews have `BookDataSource` as their data source, The first property there is `public Image State`. The images are taken from the resource file.
